Sources/XCHammer/XcodeTarget.swift

```diff
@@ -599,8 +602,14 @@ public class XcodeTarget: Hashable, Equatable {
         }
 
         // Product Name
+
         settings.productName <>= self.bundleName.map { First($0) }
 
+        if settings.productName == nil {
+            settings.productName = First(self.extractBuiltProductName())
```

Comment:
Would this code path ever be hit? It doesn't seem like it if you set the value in the initializer?

Reply:
it will now that there isn't a default value